room/runtime/api/api_lint.ignore

```diff
@@ -7,8 +7,14 @@ BuilderSetStyle: androidx.room.RoomDatabase.Builder#allowMainThreadQueries():
     Builder methods names should use setFoo() / addFoo() / clearFoo() style: method androidx.room.RoomDatabase.Builder.allowMainThreadQueries()
 BuilderSetStyle: androidx.room.RoomDatabase.Builder#createFromAsset(String):
     Builder methods names should use setFoo() / addFoo() / clearFoo() style: method androidx.room.RoomDatabase.Builder.createFromAsset(String)
+BuilderSetStyle: androidx.room.RoomDatabase.Builder#createFromAsset(String, androidx.room.RoomDatabase.PrepackagedCallback):
```

Comment:
You should not be adding these to the ignore list. Use `@Suppress` instead. We added some notes in `Contributing.md` for this.

Reply:
I tried using annotations but `./gradlew updateApi` task was still failing on metalava lint. Adding those manually fixed the issue. But I'll double check this.

Reply:
Try `@SupressLint`, take a look at `createFromInputStream` which added the same suppression.

Reply:
I think the problem is that for example `createFromFile` needs 2 suppressions: `BuilderSetStyle` and `StreamFiles`. If I add only one then it's picked up by the tooling but it still complains about second issue. When I do `@SuppressLint("BuilderSetStyle, StreamFiles")` then it doesn't work at all, same effect as not adding any suppression. I think this should work since the value of `SuppressLint` is an array of strings.

Reply:
I'm not sure it's possible to pass an array of string via varargs to the annotation? Have you tried:
`@SuppressLint({"BuilderSetStyle", "StreamFiles"})`? This is how we do it elsewhere in the codebase.

Reply:
Oh, I just noticed you are passing a single string that's comma separated :smiley: 

Definitely use `@SuppressLint({"BuilderSetStyle", "StreamFiles"})` instead.

Reply:
Ah Kotlin habits.. You're right! It worked, thanks :)